proposed/2022/dotnet-nuget-why-proposal.md

```diff
@@ -0,0 +1,140 @@
+# dotnet nuget why command
+
+- Status: **Draft**
```

Comment:
The Status field made sense (kind of) when we used to put specs in the wiki. However, now that they're markdown files in the repo, that use pull requests, I think we should remove this.

Similarly, when this PR is approved and merged, it's no longer "proposed" (file path), it's approved.  In my opinion, there should be no `proposed/` folder in the repo root.

Reply:
> The Status field made sense (kind of) when we used to put specs in the wiki. However, now that they're markdown files in the repo, that use pull requests, I think we should remove this.
> 
Agreed, will remove this. 
> Similarly, when this PR is approved and merged, it's no longer "proposed" (file path), it's approved. In my opinion, there should be no `proposed/` folder in the repo root.

I think this is probably a larger conversation, but I do agree with the reasoning.